tests/options/pugClassLocation/formatted-after.pug

```diff
@@ -0,0 +1,21 @@
+div(type="button", class=boolean ? "test" : "fact").before-class.class= "type"
+  div(type="button", class=boolean ? "test" : "fact").class-before.class.class-before.after-class= "type"
```

Comment:
```suggestion
  div(type="button", class=boolean ? "test" : "fact").class.class-before.after-class= "type"
```

Somehow it seems that here the `class-before` would get duplicated. Could you please check and fix this?

Reply:
it is actually a straight up bug with the code and test. it was dropping a class on the top two divs. working on it now.